website/src/data/users.tsx

```diff
@@ -778,6 +778,14 @@ const Users: User[] = [
     source: 'https://github.com/Hashnode/support',
     tags: ['opensource'],
   },
+  {
+    title: 'Hasura',
+    description: 'The fastest way to create a GraphQL API from your data with authorization, real-time subscriptions, and more for free.',
+    preview: require('./showcase/hasura.png'),
+    website: 'https://hasura.io/docs/',
+    source: 'https://github.com/hasura/graphql-engine/tree/master/docs',
+    tags: ['opensource', 'product', 'large'],
```

Comment:
```suggestion
    tags: ['favorite', 'opensource', 'product', 'large'],
```

Thanks

As suggested to Nicolas we can add it to favorite sites as it's a great one and have a few cool features to showcase

Reply:
I have signed the CLA, Can you please approve my PR

Reply:
we have to wait for the CLA bot to confirm, it is usually fast but sometimes take longer